app/services/hacker_news.rb

```diff
@@ -0,0 +1,38 @@
+require 'json'
+
```

Comment:
Class use to create story records if not created already from previous call to HN api and returns list of stories in the order